Add install and typescript to README

bbugh opened the pull request:
I was looking for a plugin like this and just came across it moments ago, how fortuitous for me that TypeScript was just merged with #16 a couple of hours ago! It's working great so far.

I couldn't find a way to make this work without TypeScript complaining about types, and found [a thread](https://phaser.discourse.group/t/how-to-use-plugins-in-phaser-with-typescript/10778/9) about a different plugin which was helpful. 

I added some instructions to the README for installation and for usage with TypeScript.

@zewa666 maybe you want to take a look too in case I missed something?

zewa666 commented on README.md:
````diff
@@ -15,6 +15,37 @@ Each player object contains direction and button actions. These are updated by t
+npm install phaser3-merged-input
+```
+
+Then add the plugin to Phaser 3's configuration:
````
I would make clear that this is the `global` phaser conf. you could alternatively add this as scene plugin using `this..load.scenePlugin("mergedInput");`

bbugh replied:
Fixed that, thank you. I am new to Phaser's plugins.

zewa666 commented:
oh thats a good point. wasnt thinking about it @bbugh as I'm so used to TypeScript shenanigans but certainly helpful as a quickstart

bbugh commented:
I made the recommended change and clarified the usage with TypeScript. I also fixed the example code formatting, I noticed that the tab stop is 4 in the readme, so the new example code I added is also 4 now.

GaryStanton commented:
Thank you both so much for taking the time to work on this. Sorry I'm not so quick at merging pull requests, but I can't tell you how much I appreciate you spending time on the project.
